### ISSUE TITLE
(Google Drive) Update to "Drive For Desktop"

### ISSUE THREAD

**Issue by BlythMeister** [ferventcoder/chocolatey-packages#332]
Google has released a new drive for desktop application (https://www.google.com/drive/download/)

Can the Google drive package be updated to use this installer?

**Comment by jacktose** [comment]
Agreed. Also, we should deprecate [google-backup-and-sync](https://chocolatey.org/packages/google-backup-and-sync) and [google-drive-file-stream](https://chocolatey.org/packages/google-drive-file-stream).
